style.css

```diff
@@ -0,0 +1,803 @@
+    width: 55%;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 15px;
+  }
+
+  input {
+    width: 447px;
+    height: 44px;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+    background: white;
+    font-weight: 100;
+    font-size: 20px;
+    line-height: 20px;
+    color: #172b4d;
+  }
+
+  input::placeholder {
+    color: #172b4d;
+    font-size: 17px;
+    padding-left: 10px;
+  }
+
+  textarea {
+    width: 447px;
+    height: 170px;
+    background: white;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+    margin-bottom: 0;
+    text-align: start;
+    font-size: 20px;
+  }
+
+  textarea::placeholder {
+    position: absolute;
+    color: #b3b3b3;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    padding: 10px;
+  }
+
+  .contact button {
+    margin-left: 150px;
+  }
+}
+
+/* Mobile part */
+
+@media (max-width: 767px) {
+  .desk_work,
+  .nav_links a {
+    display: none;
+  }
+
+  /* header section css */
+
+  header {
+    background-color: #fff;
+    background-image: url(./images/header-mobile.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 718px;
+    border: 1px solid #fff;
+    border-bottom-left-radius: 70px;
+    margin-bottom: 60px;
+  }
+
+  nav i {
+    font-size: 30px;
+    color: #6070ff;
+    padding-right: 40px;
+  }
+
+  nav {
+    position: fixed;
+    top: 0;
+    z-index: 5;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #fff;
+    width: 100%;
+  }
+
+  .logo h4 {
+    width: 76px;
+    height: 20px;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+    margin-left: 10px;
+  }
+
+  .present {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    padding-left: 10px;
+    gap: 10px;
+  }
+
+  .present h1 {
+    position: relative;
+    top: 200px;
+    max-width: 327px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+    margin: 0;
+  }
+
+  .present p {
+    position: relative;
+    top: 200px;
+    max-width: 627px;
+    height: auto;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    margin: 0;
+  }
+
+  .present h3 {
+    position: relative;
+    top: 200px;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #7f8cff;
+    margin: 0;
+  }
+
+  .social {
+    margin-bottom: 10px;
+    margin-left: 10px;
+    display: flex;
+    gap: 18px;
+  }
+
+  .present > .social {
+    position: relative;
+    top: 200px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .social i {
+    font-size: 30px;
+    color: #344563;
+  }
+
+  .mob_work {
+    display: flex;
```

Comment:
Well done on this section ✨ `flexbox` is a great tool but in order to make it better and easier to adapt to larger screens the main container of the projects should have a `display: grid;`  🧮 . If you have any questions about that, tag me in the comments 🗨️

Reply:
this is not a rquirement !!

Reply:
I don't understand why grid here 
mob_work class containt all item and it can't be a grid here

Reply:
It can be a `grid` with one column,  you have two sections for this it seems. One for mobile and one for Desktop, this is not bad but with a `grid` you could handle it with just one @touredri. And this is a requirement, sorry. 😿

Reply:
[Mandatory]

As mentioned in the project requirements and by the first reviewer, kindly style the works section with `grid` and not with `flex`.

![image](https://user-images.githubusercontent.com/83596135/218836302-88a10aaa-9ead-4d18-870e-82011defaacf.png)